Source/Core/VideoCommon/Fifo.cpp

```diff
@@ -315,6 +318,8 @@ void FifoManager::RunGpuLoop(Core::System& system)
           auto& fifo = command_processor.GetFifo();
           command_processor.SetCPStatusFromGPU(system);
 
+          profiler.SetMarker("GPU Loop/FIFO Loop");
```

Comment:
This should be set all the way at the top of this function. Right after AsyncRequest. Otherwise you miss `m_use_deterministic_gpu_thread`